TestApp/DeviceAgentUnitTests/CBXUITestServerTest.m

```diff
@@ -43,4 +45,15 @@ - (void)testInitPrivateSetsServerAndRegistersRoutes {
     expect(self.testServer.isFinishedTesting).to.equal(NO);
 }
 
+-(void)testInitSetsDefaultPort {
+    expect(self.testServer.server.port).equal(CBX_DEFAULT_SERVER_PORT);
+}
+
+-(void)testPassingPortFromEnvironment {
+    id processInfoMock = OCMClassMock([NSProcessInfo class]);
+    OCMStub([[processInfoMock environment] objectForKey: @"CbxServerPort"]).andReturn("41799");
+    
+    expect(self.testServer.server.port).equal(41799);
+}
```

Comment:
Use OCMExpect instead of OCMStub - I think that is possible.  And always end the test method with `OCMVerifyAll(< mocked object>)` to ensure all mocked methods were called.

You need to mock the return value of `environment`.

```
# There are probably syntax errors here...
id processInfoMock = OCMClassMock([NSProcessInfo class]);
NSDictionary *mockedEnv = @{ "CbxServerPort" : "41799" };
OCMExpect([processInfoMock environment]).andReturn(mockedEnv);

expect(self.testServer.server.port).equal(41799);

OCMVerifyAll(processInfoMock);
```